dspace_import.py

```diff
@@ -911,6 +913,20 @@ def import_handle_with_url():
 
     log("Handles with url were successfully imported!")
 
+def import_tasklistitem():
+    """
+     Import data into database.
+     Mapped table: tasklistitem
+     """
+    global workflowitem_id, eperson_id
+    json_a = read_json("tasklistitem.json")
+    for i in json_a:
+        params = {'epersonUUID': eperson_id[i['eperson_id']], 'workflowitem_id': workflowitem_id[i['workflow_id']]}
```

Comment:
Maybe add params into `try` clause because if something will be missing in the json the migration will be stopped